src/main/java/com/redhat/devtools/intellij/common/utils/ConfigHelper.java

```diff
@@ -82,4 +85,36 @@ public static Context getCurrentContext() {
             return null;
         }
     }
+
+    /**
+     * Returns {@code true} if both given contexts are equal. They are considered equal if they're equal in
+     * <ul>
+     *     <li>cluster</li>
+     *     <li>user</li>
+     *     <li>current namespace</li>
+     *     <li>extensions</li>
+     * </ul>
+     *
+     *
+     * @param thisContext
+     * @param thatContext
+     * @return true if both contexts are equal
+     *
+     * @see NamedContext
+     * @see Context
+     */
+    public static boolean areEqual(NamedContext thisContext, NamedContext thatContext) {
+        if (thisContext == null) {
+            return thatContext == null;
+        } else if (thatContext == null) {
+            return false;
+        }
+
+        return Objects.equals(thisContext.getContext(), thatContext.getContext());
+    }
+
+    public static boolean areEqual(List<NamedContext> these, List<NamedContext> those) {
```

Comment:
Just wondering what is lists don't have the same ordering ? Shouldn't it be a Collection ?

Reply:
Good point to allow comparing the more general collection type. Ordering shouldn't matter: ```AbstractCollection#equals``` verifies it via ```#containsAll```